### PR TITLE
Fetch grpc and protobuf Haskell libraries from Hackage

### PR DESCRIPTION
**Pull request by cocreature**
All the changes that resulted in us fetching them from git, have been
included in the latest Hackage releases.

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Add a line to the [release notes](https://github.com/digital-asset/daml/blob/master/unreleased.rst), if appropriate
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -555,6 +533,9 @@ hazel_repositories(
         {
             "z": "@com_github_madler_zlib//:z",
             "bz2": "@bzip2//:bz2",
+            "grpc": "@com_github_grpc_grpc//:grpc",
```
Comment (aherrmann-da) [inline WORKSPACE]: Does this work in GHCi and on macOS? It looks like this is no longer using `fat_cbits`...

Comment (cocreature) [reply WORKSPACE]: Good question, it seemed to work on ghci on Linux but I might have had old files lying around. Let’s see what CI says.

Comment (cocreature) [reply WORKSPACE]: Looks like MacOS failed. I thought it also failed on Linux in the past so I’m a bit surprised but for now, I’ll go back to custom build files here.

File: WORKSPACE [hunk 2]
```diff
@@ -570,12 +559,29 @@ hazel_repositories(
...
-                "6a803b1655824e5bec2c518b39b6def438af26135d631b60c9b70bf3af5f0db2",
+                "0.4.0.0",
+                "216fb8b5d92afc9df70512da2331e098e926239efd55e770802079c2a13bad5e",
+            ) +
+            # Not in stackage
+            hazel_hackage(
+                "grpc-haskell-core",
+                "0.0.0.0",
+                "087527ec3841330b5328d123ca410901905d111529956821b724d92c436e6cdf",
+            ) +
```
Comment (aherrmann-da) [inline WORKSPACE]: Shouldn't this one be removed then?

Comment (cocreature) [reply WORKSPACE]: oops, good catch!

Comment (cocreature) [reply WORKSPACE]: fixed.

**Comment by cocreature** [comment]
Checked ghci on MacOS locally and it seems to work.

**Pull request merged**